### PR TITLE
This strips the graphite-specific middleware

### PR DESCRIPTION
**Pull request by dannon**
Statistic functionality is provided by the statsd middleware, which should be usable for all deployments, and we don't need two separate mechanisms to maintain.

xref #6011 

My hunch is that @erasche is likely right in that they were the only person once using this (though not anymore), so I think it's safe to drop this in dev and add to the 18.05 deprecations.

### CODE REVIEW

File: config/galaxy.yml.sample [hunk 1]
```diff
@@ -52,8 +52,7 @@ uwsgi:
   # --daemon and/or production deployments.
   master: false
 
-  # Path to the application's Python virtual environment. If using Conda for
-  # Galaxy's framework dependencies (not tools!), do not set this.
+  # Path to the application's Python virtual environment.
```
Comment (nsoranzo) [inline config/galaxy.yml.sample]: This is reverting the change introduced with commit https://github.com/galaxyproject/galaxy/commit/a88e19064a38415d867f4c118dd6938d8e368632 , which should instead have been applied also to `lib/galaxy/webapps/config_manage.py` .

**Review by hexylena — APPROVED** [review]

File: config/galaxy.yml.sample (hunk 1 above)
Comment (dannon) [reply config/galaxy.yml.sample]: @nsoranzo I did a config rebuild, that's what overwrote that change I think.  I'll apply that change correctly, I guess, and then rebuild again in this branch.

Comment (dannon) [reply config/galaxy.yml.sample]: Done in the next two commits.  Thanks for noticing this, @nsoranzo!

Comment (nsoranzo) [reply config/galaxy.yml.sample]: That's exactly what I meant, thanks!

**Review by nsoranzo — APPROVED** [review]

**Pull request merged**